packages/rs-drive/src/drive/document/delete/remove_document_from_primary_storage/v0/mod.rs

```diff
@@ -24,7 +25,7 @@ impl Drive {
     #[inline(always)]
     pub(super) fn remove_document_from_primary_storage_v0(
         &self,
-        document_id: [u8; 32],
+        document_id: Identifier,
```

Comment:
nit: have these `[u8; 32]` -> `Identifier` in a separate commit to improve reviewability

Reply:
Indeed, but I'm lazy ><